.github/actions/approve-bot/helpers.js

```diff
@@ -33,11 +33,14 @@ function allChangelogLinesAreVersionBump(changelogLines) {
   )
 }
 
-function isPointlessGatsbyBump(body) {
+function isPointlessVersionBump(body) {
+  const pointlessBumpLinks = [
+    'https://github.com/gatsbyjs/gatsby',
+    'https://github.com/typescript-eslint/typescript-eslint',
+  ]
+
   const lines = body.split(/\r?\n/)
-  if (
-    !lines[0].includes('https://github.com/gatsbyjs/gatsby') // lgtm [js/incomplete-url-substring-sanitization]
-  ) {
+  if (!pointlessBumpLinks.some(link => new RegExp(link).test(lines[0]))) {
```

Comment:
Any reason why you've used a regex here rather than the previous `includes` match? The `pointlessBumpLinks` elements are not escaped, the `/` even make them invalid I believe.

Reply:
@PyvesB apologies, I didn't know the `includes` match matches partial tokens as well. I will revert that part, thanks!